### ISSUE TITLE
🐜 Accessiblity Bug: Work Experience Start data/End Date Field 

### ISSUE THREAD

**Issue by RM-1978** [GCTC-NTGC/gc-digital-talent#5415]
## ❌ Current behaviour (issue)
Date Form label is attached to the unsaved changes The start date and end date fields are not labeled when filling out the form. Screen Reader user must go into navigational mode to find the labels.
## Location: 
Skills and Experience,:  Work Experience, Education, Community, Personal and Award forms before the skill picker. 
### AT configuration: 
VoiceOver, IOS, Safari 
### How to reproduce
1. Navigate using your keyboard to a work experience entry page using VoiceOver.  
2. Check label readout for dates. 
3. Enter the input field. 
4. Choose a date based on what you hear.
5.  Run the same test with JAWS, Windows 11, chrome
### Fable Video
[Skills and experience date input challenge](https://app.makeitfable.com/clip/63c9ed0bd3499e0081ebf762)
## ✔ Expected behaviour (fix)
- Form Labels should be read out clearly before entering the input field. Interaction with the text entry vs the date spin box should be clear.
### Acceptance Criteria
- [ ] Date field form labels are read out properly. 
- [ ] Field Input method (text entry vs spin box) is clear to screen reader users.

**Comment by petertgiles** [comment]
Hi Rosalie.  Can we drill into this one?  I watched the video and could see that the tester was struggling but it's not clear to me which remediations we need to make.

For the first point, what change should be made to the form label?  When I check the a11y tree in Firefox it appears that the input is properly related to the label.
![image](https://user-images.githubusercontent.com/8978655/213734658-db78f82c-f207-4e2c-b711-e9691a6a1c50.png)

For the second point you've indicated that the field input method should be clear, but what change needs to be made to make it clear?

**Comment by RM-1978** [comment]
Hi @petertgiles, thanks for this note. From what I can tell, the real issue here is if the field component we want here is a formatted text entry box (mm/dd/yyyy) or a date picker (Spin box or 3 spin boxes). I think that all users who identified this issue (2 compatibility testers, 1 interview person and3 self-test participants) recommend that we choose one component or the other and make sure that it is clear to the any user how they enter dates (either through the labels or new instructions).

**Comment by petertgiles** [comment]
OK, back to the date picker.  :smile:  OK, I've added a placeholder issue to the board and I'm going to block this on that.

**Comment by esizer** [comment]
Was this solved by the new date picker?

**Comment by petertgiles** [comment]
Yeah, Lee's requested solution  on Feb 7 was to reimplement the date picker.  This is completed.
